.generated_docs

```diff
@@ -23,12 +23,14 @@ docs/man/man1/kubectl-convert.1
 docs/man/man1/kubectl-create.1
 docs/man/man1/kubectl-delete.1
 docs/man/man1/kubectl-describe.1
+docs/man/man1/kubectl-drain.1
 docs/man/man1/kubectl-edit.1
 docs/man/man1/kubectl-exec.1
 docs/man/man1/kubectl-explain.1
 docs/man/man1/kubectl-expose.1
 docs/man/man1/kubectl-get.1
 docs/man/man1/kubectl-label.1
+docs/man/man1/kubectl-lame.1
```

Comment:
"disable", "deactivate", "decommission", etc. have the problem that they imply complete removal from service.

How about `cordon` and `evacuate` rather than `lame` and `drain`?

@mml Where was the previous discussion thread about the names? Was it on #17393?